src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.InvalidUserDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userIsNull_notOk() {
+        Exception exception = assertThrows(InvalidUserDataException.class, () -> {
+            registrationService.register(null);
+        });
+        assertEquals("User cannot be null", exception.getMessage());
+    }
+
+    @Test
+    void register_userWithExistingLogin_notOk() {
+        User user1 = new User();
+        user1.setLogin("testuser");
+        user1.setPassword("password");
+        user1.setAge(20);
+        registrationService.register(user1);
```

Comment:
You should not use the `register` method to add a user for testing, because we cannot guarantee its correctness yet. Use low-level methods like `Storage.people.add()` directly instead.